### PR TITLE
Error in upstream corrected

### PR DESCRIPTION
**Pull request by nohan-antony**

### CODE REVIEW

File: src/ui/src/app/dashboard/dashboard-body/main/employee-directory/employee-directory.component.html [hunk 1]
```diff
@@ -70,7 +70,10 @@ <h5 class="card-title">{{employee.firstName}} {{employee.lastName}}</h5>
         </div>
 
         <div class="w-100">
-          <mat-paginator [length]="employeesPagecount.totalPages" [pageSize]="pageSize" [pageIndex]="pageNumber"
+          {{employeesPagecount.totalPages}}
```
Comment (eldhopjoy) [inline src/ui/src/app/dashboard/dashboard-body/main/employee-directory/employee-directory.component.html:73]: remove this

File: src/ui/src/app/dashboard/dashboard-body/main/employee-directory/employee-directory.component.html [hunk 2]
```diff
@@ -70,7 +70,10 @@ <h5 class="card-title">{{employee.firstName}} {{employee.lastName}}</h5>
...
+          {{pageSize}}
+          {{pageNumber}}
```
Comment (eldhopjoy) [inline src/ui/src/app/dashboard/dashboard-body/main/employee-directory/employee-directory.component.html:75]: these also

File: src/ui/src/app/dashboard/dashboard-body/main/employee-directory/employee-directory.component.ts [hunk 3]
```diff
@@ -69,6 +70,8 @@ export class EmployeeDirectoryComponent implements OnInit {
       next: (data) => {
         console.log(data)
         this.isLoading = false; 
+        if(this.paginator1)
+        this.paginator1.length=data.totalPages;
```
Comment (eldhopjoy) [inline src/ui/src/app/dashboard/dashboard-body/main/employee-directory/employee-directory.component.ts:74]: not need of this paginator1

File: src/ui/src/app/dashboard/dashboard-body/main/employee-directory/employee-directory.component.ts [hunk 4]
```diff
@@ -69,6 +70,8 @@ export class EmployeeDirectoryComponent implements OnInit {
       next: (data) => {
         console.log(data)
```
Comment (eldhopjoy) [inline src/ui/src/app/dashboard/dashboard-body/main/employee-directory/employee-directory.component.ts:71]: remove console logs

**Review by Aravindh-sreenivas — APPROVED** [review]

File: src/ui/src/app/user/check-user/check-user.component.ts [hunk 5]
```diff
@@ -33,16 +33,16 @@ export class CheckUserComponent {
     ;
     if(this.userForm.valid){
      
-      this.userService.getEmployeeSignUp(this.userForm.value.employeeCode,this.userForm.value.rfId).subscribe({
+      this.userService.getEmployeeSignUp(this.userForm.value.employeeCode,this.userForm.value.rfid).subscribe({
```
Comment (eldhopjoy) [inline src/ui/src/app/user/check-user/check-user.component.ts:36]: camelCase, there is a issue yesterday , so please correct on the form also

File: src/ui/src/app/dashboard/dashboard-body/main/employee-directory/employee-directory.component.ts (hunk 3 above)
Comment (nohan-antony) [reply src/ui/src/app/dashboard/dashboard-body/main/employee-directory/employee-directory.component.ts]: done

File: src/ui/src/app/dashboard/dashboard-body/main/employee-directory/employee-directory.component.ts (hunk 4 above)
Comment (nohan-antony) [reply src/ui/src/app/dashboard/dashboard-body/main/employee-directory/employee-directory.component.ts:74]: okay

File: src/ui/src/app/user/check-user/check-user.component.ts (hunk 5 above)
Comment (nohan-antony) [reply src/ui/src/app/user/check-user/check-user.component.ts:36]: done

**Review by eldhopjoy — APPROVED** [review]

**Pull request merged**